tests/constants/label.constants.ts

```diff
@@ -42,7 +42,8 @@ export const DARK_BG_LIGHT_TEXT = {
   cursor: 'default',
   padding: '3px',
   color: `#${COLOR_WHITE}`,
-  'font-weight': '410'
+  'font-weight': '410',
+  display: 'inline-flex'
```

Comment:
There's a bit of redundancy here, as we're defining the same CSS multiple times.

I think the common CSS properties (e.g. padding, font-weight, display) can be defined just once in LabelService. But let's take this up in another PR.

Reply:
Yup, I think we can create a new issue for this. Think it can be a good first issue for new developers too since its just defining the common attributes that are to be reused

Reply:
@zatkiller can help to create the issue for this? Thanks!

Reply:
Hey @anubh-v, sure I will create it later